<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QTabWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1470</width>
    <height>832</height>
   </rect>
  </property>
  <widget class="QWidget" name="Invoices">
   <attribute name="title">
    <string>Page</string>
   </attribute>
   <layout class="QVBoxLayout">
    <item>
     <widget class="QPushButton" name="save_btn">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
      <property name="class" stdset="0">
       <string>primary outlined</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollarea_qwidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1432</width>
         <height>755</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="scrollarea_qwidget_layout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignTop">
         <widget class="QWidget" name="frame" native="true">
          <layout class="QVBoxLayout" name="frame_verticalLayout">
           <item alignment="Qt::AlignmentFlag::AlignTop">
            <widget class="QWidget" name="frame_invoice" native="true">
             <layout class="QVBoxLayout" name="frame_invoice_layout">
              <item>
               <widget class="QLabel" name="label_new_invoice">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>New Invoice</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_company">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                  <fontweight>DemiBold</fontweight>
                 </font>
                </property>
                <property name="text">
                 <string>Company</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_subtitle">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(154, 153, 150)</string>
                </property>
                <property name="text">
                 <string>Change the address, logo, and other information for your company.</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget" native="true">
                <layout class="QHBoxLayout" name="_2">
                 <property name="spacing">
                  <number>6</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QWidget" name="frame_title_sub" native="true">
                   <property name="minimumSize">
                    <size>
                     <width>250</width>
                     <height>250</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>400</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLineEdit" name="title_input">
                      <property name="minimumSize">
                       <size>
                        <width>250</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="placeholderText">
                       <string>Title</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="subheading_input">
                      <property name="minimumSize">
                       <size>
                        <width>250</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="placeholderText">
                       <string>Subheading</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="frame_btn_logo" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>250</width>
                     <height>250</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>400</width>
                     <height>250</height>
                    </size>
                   </property>
                   <property name="baseSize">
                    <size>
                     <width>250</width>
                     <height>250</height>
                    </size>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QPushButton" name="select_logo_btn">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>250</width>
                        <height>250</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>250</width>
                        <height>250</height>
                       </size>
                      </property>
                      <property name="baseSize">
                       <size>
                        <width>250</width>
                        <height>250</height>
                       </size>
                      </property>
                      <property name="font">
                       <font>
                        <pointsize>18</pointsize>
                        <bold>true</bold>
                       </font>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="icon">
                       <iconset theme="QIcon::ThemeIcon::InsertImage"/>
                      </property>
                      <property name="iconSize">
                       <size>
                        <width>128</width>
                        <height>128</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_logo">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>250</width>
                     <height>250</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>250</width>
                     <height>250</height>
                    </size>
                   </property>
                   <property name="baseSize">
                    <size>
                     <width>250</width>
                     <height>250</height>
                    </size>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::Shape::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Shadow::Raised</enum>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_2">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="image_lable">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>250</width>
                        <height>250</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>250</width>
                        <height>250</height>
                       </size>
                      </property>
                      <property name="baseSize">
                       <size>
                        <width>250</width>
                        <height>250</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="frame_address" native="true">
                   <property name="minimumSize">
                    <size>
                     <width>250</width>
                     <height>250</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>400</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_3">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLineEdit" name="name_input">
                      <property name="minimumSize">
                       <size>
                        <width>250</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>35</height>
                       </size>
                      </property>
                      <property name="placeholderText">
                       <string>Name</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="email_input">
                      <property name="minimumSize">
                       <size>
                        <width>250</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>35</height>
                       </size>
                      </property>
                      <property name="placeholderText">
                       <string>Email</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="tax_number_input">
                      <property name="minimumSize">
                       <size>
                        <width>250</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>35</height>
                       </size>
                      </property>
                      <property name="placeholderText">
                       <string>Tax Number</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="phone_input">
                      <property name="minimumSize">
                       <size>
                        <width>250</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>35</height>
                       </size>
                      </property>
                      <property name="placeholderText">
                       <string>Phone</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="address_input">
                      <property name="minimumSize">
                       <size>
                        <width>250</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>35</height>
                       </size>
                      </property>
                      <property name="placeholderText">
                       <string>Address</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="country_input">
                      <property name="minimumSize">
                       <size>
                        <width>250</width>
                        <height>30</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>35</height>
                       </size>
                      </property>
                      <property name="placeholderText">
                       <string>Country</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="frame_billing" native="true">
             <layout class="QVBoxLayout" name="frame_billing_layout">
              <item>
               <widget class="QLabel" name="label_billing">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Billing</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_3" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLineEdit" name="billing_name_input">
                   <property name="minimumSize">
                    <size>
                     <width>250</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>400</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="placeholderText">
                    <string>Name</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="price_input">
                   <property name="minimumSize">
                    <size>
                     <width>250</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>400</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="placeholderText">
                    <string>Price</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="count_input">
                   <property name="minimumSize">
                    <size>
                     <width>250</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>400</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="placeholderText">
                    <string>Count</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="add_to_record_btn">
                   <property name="minimumSize">
                    <size>
                     <width>200</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Add to record</string>
                   </property>
                   <property name="class" stdset="0">
                    <string>primary outlined</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QTableView" name="tableView">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>350</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item alignment="Qt::AlignmentFlag::AlignTop">
            <widget class="QWidget" name="frame_advanced" native="true">
             <layout class="QVBoxLayout" name="frame_advanced_layout">
              <item>
               <widget class="QLabel" name="label_advanced">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Advanced</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="Customers">
   <attribute name="title">
    <string>Page</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
